test/run-drun/stable-region-blob.mo

```diff
@@ -0,0 +1,190 @@
+//MOC-FLAG --stable-regions
+
```

Comment:
The name suggest this tests blob load/store, but those bits are commented out and its just testing scalar read write.